package.json

```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "js-yaml": "^3.9.0",
-    "kaitai-struct": "^0.8.0-SNAPSHOT.2",
-    "kaitai-struct-compiler": "^0.7.1"
+    "kaitai-struct": "^0.9.0-SNAPSHOT.1",
+    "kaitai-struct-compiler": "^0.8.0-SNAPSHOT.20180205.11616"
```

Comment:
It's kind of weird: if we're about using 0.8 compiler, then we need to use 0.8 runtime as well?

And, on top of that, likely we'd want release versions, not snapshot versions?

Reply:
yeah probably true on both accounts :)

I just bumped to the latest version of everything, as in, these are the version I would have installed if I wasn't using this loader and just ran `yarn add kaitai-struct` and `kaitai-struct-compiler`